examples/react-app/package.json

```diff
@@ -20,6 +25,11 @@
     "@types/react": "18.2.71",
     "@types/react-dom": "18.2.22",
     "@vitejs/plugin-react": "4.2.1",
+    "prettier": "^3.1.1",
+    "prettier-plugin-tailwindcss": "0.5.12",
```

Comment:
By removing the prettier configs also remove this deps and any other deps that are not in use.